src/mcts/search.cc

```diff
@@ -138,6 +139,19 @@ void Search::SendUciInfo() REQUIRES(nodes_mutex_) {
       uci_info.pv.push_back(iter.GetMove(flip));
       if (!iter.node()) break;  // Last edge was dangling, cannot continue.
     }
+
+    // Mate display if certain win (or loss) with distance to mate set to
+    // length of pv (average  mate).
+    // If win is based on propagated TB bit, length of mate is
+    // adjusted by +1000; If root filtered TB moves are draw display 0.
+    if (params_.GetCertaintyPropagation()) {
+      if (edge.IsCertain() && edge.GetEQ() != 0) {
+        uci_info.mate = edge.GetEQ() * ((uci_info.pv.size() + 1) / 2 +
+                                        (edge.IsPropagatedTBHit() ? 1000 : 0));
+      } else if (root_syzygy_rank_ == 1) {
+        uci_info.score = 0;
```

Comment:
@Videodr0me It looks like @jjoshua2 is using something based on this PR for Leelenstein at CCC, and it's now reporting `cp 0` for tablebase positions even when they're won positions. Fortunately the opponent knows it's a losing position, so CCC doesn't adjudicate draw. I'm guessing the return value of `PopulateRootMoveLimit` is not what you think it is.

Reply:
In the mean time, the simplest fix I can recommend to @jjoshua2 is to comment out this `uci_info.score = 0;` line as it'll just revert to the normal score it would have shown.